README.md

```diff
@@ -126,6 +126,13 @@ Run the command:
 $ make acc-test
 ```
 
+To run the acc-test specific to a resource make use of the build-tags.
```

Comment:
- You can mention that since we have the build tags in place, if anybody tries to run tests without specifying any tags then ideally no tests are run. So the only way to run them is by specifying the corresponding tags.
- For better understanding add a point that the build tags are nothing but the corresponding resource names

Reply:
Done

Reply:
- The second point is still missing, you can specify the build tags are nothing but corresponding resource names.

- Also since by default all tags are included in BUILD_TAGS, you can mention user can use the `make test-tags` target to run all acc tests by not setting BUILD_TAGS manually to selected resources.

Reply:
Done